projects/manage-supervision-and-delius/src/main/kotlin/uk/gov/justice/digital/hmpps/service/RiskService.kt

```diff
@@ -33,14 +34,44 @@ class RiskService(
     fun getPersonRiskFlags(crn: String): PersonRiskFlags {
         val summary = personRepository.getSummary(crn)
         val riskFlags = riskFlagRepository.findByPersonId(summary.id)
+        val opd = nsiRepository.findByPersonIdAndTypeCode(summary.id, NsiType.Code.OPD_COMMUNITY_PATHWAY.value)
```

Comment:
Do we need to check the NSI active flag here?

Reply:
yes, changed